dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -957,7 +957,7 @@ const shared_ptr<vector<shared_ptr<ComponentInterface> > >
 DataInterfaceIIDM::findConnectedComponents() {
   auto connectedComponents = boost::make_shared<vector<shared_ptr<ComponentInterface> > >();
   for (auto& component : components_) {
-    if ((component.second)->isPartiallyConnected())
+    if ((component.second)->isConnected())
```

Comment:
I'm not sure about this one. If a line is partially connected then the equipment should be seen as a lost one no?

Reply:
Yes you are right (even if it can be a matter of point of view), but actually the development that modify it broke some unittests that were not being run. So I propose to make the unittests work again and chnage the behavior in another PR

Reply:
ok!

Reply:
I will reopen the old issue or create a new one to make sure we do it